crates/astria-auctioneer/src/auctioneer/mod.rs

```diff
@@ -272,6 +283,9 @@ impl Auctioneer {
             "block_hash",
             field::display(executed_block.sequencer_block_hash()),
         );
+
+        self.metrics.increment_executed_blocks_received_counter();
```

Comment:
this can act as a proxy for the metric "number of auctions conducted" since an auction starts when the auctioneer receives an executed block.